examples/tutorials/advanced/websockets-example-MNIST/start_websocket_servers.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
```

Comment:
Q: Why not simply remove this and in all the places where this was used ad the other one -- what I understand is that we currently have 2 python files and this one calls the other one. My idea is to delete this one and change all the occurrences where we called this file to the other one

Reply:
I think we can delete this file because its folder doesn't contain a notebook in which it is used. Now, this file is of no use. So should I delete it?